src/component/create.modal.tsx

```diff
@@ -0,0 +1,123 @@
+"use client";
+import { useState } from "react";
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+import Modal from "react-bootstrap/Modal";
+import { toast } from "react-toastify";
+import { mutate } from "swr";
+
+interface IProps {
+  showModalCreate: boolean;
+  setShowModalCreate: (v: boolean) => void;
+}
+
+function CreateModal(props: IProps) {
+  const { showModalCreate, setShowModalCreate } = props;
+  const [title, setTitle] = useState<string>("");
+  const [author, setAuthor] = useState<string>("");
+  const [content, setContent] = useState<string>("");
+
+  const handleSubmit = () => {
+    if (!content && !content && !author) {
+      toast.error("Not empty data !");
+      return;
+    }
```

Comment:
@HoVanDi - pls move this if the condition to line 38 (behind the `if (!content) {`

Reply:
Thank you for your feedback. I've moved this if condition to line 38